resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -278,4 +285,36 @@ func copy(data []byte, v interface{}) {
 	dataS := string(data)
 
 	reflect.ValueOf(v).Elem().Set(reflect.ValueOf(&dataS))
+}
+
+// process the model to remove omitempty tags on optional properties
+func processModel(model interface{}) ([]byte, error) {
+	value := reflect.ValueOf(model)
```

Comment:
Super nice.  Good job figuring out the tags API.  This might be useful later on in other parts of CX as Code